packages/service-library/src/servicelib/aiohttp/application.py

```diff
@@ -14,11 +18,35 @@ async def shutdown_info(app: web.Application):
     print(f"INFO: SHUTTING DOWN {app} ...", flush=True)
 
 
+async def stop_background_tasks(app: web.Application):
+    running_tasks = app[APP_FIRE_AND_FORGET_TASKS_KEY]
+    task: asyncio.Task
+    for task in running_tasks:
+        task.cancel()
+    try:
+        MAX_WAIT_TIME_SECONDS = 5
+        results = await asyncio.wait_for(
+            asyncio.gather(*running_tasks, return_exceptions=True),
+            timeout=MAX_WAIT_TIME_SECONDS,
+        )
+        if bad_results := list(filter(lambda r: isinstance(r, Exception), results)):
```

Comment:
So, then the idea is: you have 5 seconds to stop (I would add this as an option parameter with this default) and if you stop, we log the rrors

MINOR: 
```python
 if errors := [r for r in results if isinstance(r,Exception) ] :
    ...
```